code/datums/elements/strippable.dm

```diff
@@ -47,6 +47,9 @@
 	if(over != user)
 		return
 
+	if(isrevenant(user))
+		return
```

Comment:
This should just check if the mob uses a jaunt movement type (eminence, revenent, other jaunt moving things/ghosts like demons)

Reply:
will do